language/functional_tests/tests/testsuite/generics/builtins.mvir

```diff
@@ -0,0 +1,24 @@
+//! no-run: verifier, runtime
```

Comment:
Is there any harm in running the verifier here (similar for other tests)?

Reply:
Yes. Currently the in the verifier there's logic like "get the kind of a SignatureToken" (without the context), which panics when a type parameter is seen.